sechub-api-java/src/main/java/com/mercedesbenz/sechub/api/java/demo/cli/SechubServerTest.java

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.api.java.demo.cli;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.mercedesbenz.sechub.api.java.*;
+import com.mercedesbenz.sechub.api.java.auth.Authentication;
+import com.mercedesbenz.sechub.api.java.auth.HttpBasicAuth;
+import com.mercedesbenz.sechub.api.java.demo.cli.exceptions.OpenAPITestToolRuntimeException;
+
+public class SechubServerTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SechubServerTest.class);
+    SecHubClient secHubClient = null;
+    ApiClient apiClient = null;
+    StringBuilder fullConnectionURI = new StringBuilder();
```

Comment:
A StringBuilder is a good way to create a string in Java. Strings in Java are immutable. Meaning, while concatenating two strings one creates a third string. This can be expensive in cases in which the String contains a lot of text.

However, StringBuilder should not be use as a class attribute. You use a StringBuilder to "create/make" a new string.

```suggestion
    String fullConnectionURI = "";
```